install_config/registry/securing_and_exposing_registry.adoc

```diff
@@ -320,9 +339,8 @@ not the route name and port. See `oc get imagestreams` for details.
 In the `<host>/test/busybox` example above, `test` refers to the project name.
 ====
 
-
 [[access-secure-registry-by-exposing-route]]
-=== Exposing a Secure Registry
+=== Manually Exposing a Secure Registry
```

Comment:
It seems like this section actually is just showing how to `docker login` to a secure & exposed registry. ???